src/Server/mediator.ts

```diff
@@ -1,82 +0,0 @@
-import EventEmitter from "eventemitter3"
```

Comment:
We could remove this dependency once we get rid of https://github.com/artsy/force/blob/main/src/Utils/Events.ts — there's some legacy tracking stuff that's still using it apparently.

Reply:
How this stuff works is pretty straightforward, if you want to swap it out with some kind of built in event pub sub: https://github.dev/artsy/force/blob/main/src/Apps/Artwork/Components/ArtistInfo.tsx#L37. Obvs, no need to do that here

Reply:
And doesn't even really need to be a pub sub.. just call the analytics track event directly. It can be a direct function call, and delete this whole file

Reply:
Yeah, that's what we'd want to do just update these to use the hooks